src/main/java/org/team1540/base/motionprofiling/MotionProfile.java

```diff
@@ -13,7 +15,15 @@
   private double lastTime;
   private boolean isFinished = false;
 
+  public MotionProfile(MotionProfilingProperties... properties) {
+    realConstructor(new HashSet<>(Arrays.asList(properties)));
+  }
+
   public MotionProfile(Set<MotionProfilingProperties> motionProfiles) {
+    realConstructor(motionProfiles);
```

Comment:
Why are you also accepting a set? If people have a collection they can use the spread operator on the varargs constructor.

Reply:
Cuz it's represented internally as a set (since I don't need the order info.) Set -> array -> set seems kinda dumb.